src/rendergraph/opengl/CMakeLists.txt

```diff
@@ -62,15 +62,17 @@ target_link_libraries(rendergraph_gl PUBLIC
     Qt6::Gui
     Qt6::OpenGL
 )
-find_package(Microsoft.GSL CONFIG)
-if(Microsoft.GSL_FOUND)
-  target_link_libraries(rendergraph_gl PRIVATE Microsoft.GSL::GSL)
-else()
-  # check if the headers have been installed without cmake config (< 3.1.0)
-  check_include_file_cxx(gsl/gsl HAVE_GSL_GSL)
-  if(NOT HAVE_GSL_GSL)
-    unset(HAVE_GSL_GSL CACHE) # unset cache to re-evaluate this until it succeeds. check_include_file_cxx() has no REQUIRED flag.
-    message(FATAL_ERROR "ms-gsl development headers (libmsgsl-dev) not found")
+if(WIN32)
+  find_package(Microsoft.GSL CONFIG)
+  if(Microsoft.GSL_FOUND)
```

Comment:
why `WIN32` only? the GSL is not microsoft specific, its just created and maintained by them but it works on any compiler.

Reply:
Oh, Antoine added this, and it didn’t build on macOS so I added the if(WIN32). I guess it should be if(NOT APPLE).

Reply:
No, GSL provides general functionality, we use in Mixxx on all platforms.

Reply:
Last time I checked across platforms I own, this was needed to prevent duplicate symbols issues on Linux, but feel free to double check.

Reply:
Ok, got it. This issue is when building the examples standalone, and not having the vcpkg in my cmake paths. I'll revert this and look for a solution to build the examples.